request_logger.go

```diff
@@ -36,6 +38,22 @@ func (rw *responseWriter) WriteHeader(code int) {
 	return
 }
 
+func StatsdMiddleware(client *statsd.Client) func(http.Handler) http.Handler {
+	return func(next http.Handler) http.Handler {
+		fn := func(w http.ResponseWriter, r *http.Request) {
+
+			start := time.Now()
+			wrapped := wrapResponseWriter(w)
+			next.ServeHTTP(wrapped, r)
+			go func() {
+				client.Histogram("request.latency", float64(time.Since(start).Nanoseconds()), []string{fmt.Sprintf("status:%d", wrapped.status), fmt.Sprintf("path:%s", r.URL.EscapedPath())}, 1)
```

Comment:
is this being prefixed with the name of the service? When using mscli's statsd that happens automatically. Since you are using the sdk directly you're probably missing that. We don't want an unscoped metric like that

Reply:
No its not that is a mistake i will prefix it with admission-contorl